filters/auth/grantconfig.go

```diff
@@ -84,6 +84,9 @@ type OAuthConfig struct {
 	// on the request to a header with this name.
 	AccessTokenHeaderName string
 
+	// GrantTokeninfoKeys, optional. When not empty, keys not in this list are removed from the tokeninfo map.
+	GrantTokeninfoKeys []string
```

Comment:
Maybe `map[string]struct{}`?

Reply:
Pass it as []string and convert it to map[string]struct{} in CreateFilter ?

Reply:
Added a lookup map to the config